### PR TITLE
Week Two, Part Three

### PR DESCRIPTION
**Pull request by malachiblenman**

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -23,6 +23,13 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+  <groupId>com.google.code.gson</groupId>
```
Comment (dgalbraith33) [inline portfolio/pom.xml:28]: nit: fix the indentation here.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -29,7 +29,12 @@ function addRandomFact() {
 }
 
 function FetchDispGreeting (){
-    fetch("/data").then(response => response.text()).then((greeting) => {
-        document.getElementById("greeting").innerHTML = greeting;
+      fetch("/data").then(response => response.json()).then((messages) => {
+            var Affirmations = document.getElementById("messages");
```
Comment (dgalbraith33) [inline portfolio/src/main/webapp/script.js:33]: nit: generally for variable names we start them with lowercase. Usually only class names are started with uppercase.

File: portfolio/pom.xml (hunk 1 above)
Comment (malachiblenman) [reply portfolio/pom.xml]: Done!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (malachiblenman) [reply portfolio/src/main/webapp/script.js]: Done! Thanks for the tip

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -24,7 +24,7 @@
 html{
     font-size: 1.5rem;
     font-family: 'arial', sans-serif;
-    color:rgb(255, 254, 254);
```
Comment (dgalbraith33) [inline portfolio/src/main/webapp/style.css:27]: Any reason for the change here and below? I think its generally preferable to use the hex codes.

Comment (malachiblenman) [reply portfolio/src/main/webapp/style.css:27]: My bad, I think that was a mistake! I meant to change them all to color:white!

**Pull request merged**